fixtures/workers-with-assets/package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "assets-worker",
+	"name": "workers-assets-with-worker",
```

Comment:
nit: `workers-assets-with-user-worker` or `dynamic-workers-with-assets` and explain what that means in the readme

Reply:
prefer the uber explicit `workers-assets-with-user-worker`